test/org/sagebionetworks/bridge/models/accounts/UserSessionInfoTest.java

```diff
@@ -52,9 +59,20 @@ public void userSessionInfoSerializesCorrectly() throws Exception {
         assertEquals("foo", node.get("dataGroups").get(0).asText());
         assertEquals("staging", node.get("environment").asText());
         assertEquals("UserSessionInfo", node.get("type").asText());
+
+        JsonNode consentMap = node.get("consentStatuses");
+        
+        JsonNode consentStatus = consentMap.get("AAA");
+        assertEquals("Consent", consentStatus.get("name").asText());
+        assertEquals("AAA", consentStatus.get("subpopulationGuid").asText());
+        assertTrue(consentStatus.get("required").asBoolean());
+        assertTrue(consentStatus.get("consented").asBoolean());
+        assertFalse(consentStatus.get("signedMostRecentConsent").asBoolean());
+        assertEquals("ConsentStatus", consentStatus.get("type").asText());
+        assertEquals(6, consentStatus.size());
         
         // ... and no things that shouldn't be there
-        assertEquals(11, node.size());
+        assertEquals(12, node.size());
     }
```

Comment:
I removed all of these and the tests passed, I will run the SDK against it as well since those tests span calls and require session support.